dummy-files/main.py

```diff
@@ -0,0 +1,94 @@
+
+    temp_imu_obj = IMUSensor(frame, timestamp, location, rotation, 10.5, 10.1, 0.4)
+    temp_imu_dict = {"frame": temp_imu_obj.frame, "timestamp": temp_imu_obj.timestamp,
+                     "location": temp_imu_obj.location, "rotation": temp_imu_obj.rotation,
+                     "accelerometer": temp_imu_obj.accelerometer, "compass": temp_imu_obj.compass,
+                     "gyroscope": temp_imu_obj.gyroscope}
+
+    temp_lidar_obj = LidarSensor(frame, timestamp, location, rotation, 3, 15.5, None)
+    temp_lidar_dict = {"frame": temp_lidar_obj.frame, "timestamp": temp_lidar_obj.timestamp,
+                       "location": temp_lidar_obj.location, "rotation": temp_lidar_obj.rotation,
+                       "channels": temp_lidar_obj.channels, "horizontal_angle": temp_lidar_obj.horizontal_angle,
+                       "raw_data": temp_lidar_obj.raw_data}
+
+    temp_radar_obj = RadarSensor(frame, timestamp, location, rotation, 30.5, 9.8, 15.5, 55.7)
+    temp_radar_dict = {"frame": temp_radar_obj.frame, "timestamp": temp_radar_obj.timestamp,
+                       "location": temp_radar_obj.location, "rotation": temp_radar_obj.rotation,
+                       "altitude": temp_radar_obj.altitude, "azimuth": temp_radar_obj.azimuth,
+                       "depth": temp_radar_obj.depth, "velocity": temp_radar_obj.velocity}
+
+    temp_slidar_obj = LidarSensor(frame, timestamp, location, rotation, 2, 16.9, None)
+    temp_slidar_dict = {"frame": temp_slidar_obj.frame, "timestamp": temp_slidar_obj.timestamp,
+                        "location": temp_slidar_obj.location, "rotation": temp_slidar_obj.rotation,
+                        "channels": temp_slidar_obj.channels, "horizontal_angle": temp_slidar_obj.horizontal_angle,
+                        "raw_data": temp_slidar_obj.raw_data}
+
+    temp_dict = {
+        "battery": temp_battery_dict,
+        "camera": temp_camera_dict,
+        "gnss": temp_gnss_dict,
+        "imu": temp_imu_dict,
+        "lidar": temp_lidar_dict,
+        "radar": temp_radar_dict,
+        "s_lidar": temp_slidar_dict
+    }
+    temp_json = json.dumps(temp_dict, indent=2)
+
+    return temp_json
+
+
+def save_file(name, path1, frame, timestamp, location, rotation):
+    with open(f"{path1}/{name}.json", "w") as outfile:
+        outfile.write(dummy_data(frame, timestamp, location, rotation))
+
+
+def main(dir_path=None, dir_na=None, time_lapse=10):
+    name = "vehicle1"
+    timestamp = int(time.time())
+    if os.path.exists(dir_path):
+        try:
+            os.mkdir(dir_path + dir_na)
+            for dt in range(time_lapse):
+                save_file(name + f"-{dt}s", dir_path + dir_na, 1, dt + timestamp, 2, 3)
+        except FileExistsError:
+            print("Directory exists, create new one..")
+    else:
+        print("Directory path does not exist or is not defined.")
+
+
+if __name__ == "__main__":
+    directory_path = "C:/Users/AndreMecklenborg/Desktop/"
```

Comment:
Maybe your local path should not be included in the repo. What about using relative path that would work on all computers with this repo?

Reply:
yep that sliped into the commit